dvc/ignore.py

```diff
@@ -16,6 +16,16 @@
 logger = logging.getLogger(__name__)
 
 
+class PathStringTrie(StringTrie):
+    """Trie based on platform-dependent separator for pathname components."""
+
+    # pylint: disable=useless-super-delegation
+    def __init__(self, *args, separator=os.sep, **kwargs):
```

Comment:
Doesn't look like anyone is going to use it anyways
```suggestion
    def __init__(self, *args, **kwargs):
```